appeals/e2e/cypress/e2e/back-office-appeals/caseNotes.spec.js

```diff
@@ -0,0 +1,42 @@
+// @ts-nocheck
+/// <reference types="cypress"/>
+
+import { users } from '../../fixtures/users';
+import { CaseDetailsPage } from '../../page_objects/caseDetailsPage.js';
+import { ListCasesPage } from '../../page_objects/listCasesPage';
+import { urlPaths } from '../../support/urlPaths.js';
+import { happyPathHelper } from '../../support/happyPathHelper.js';
+
+const listCasesPage = new ListCasesPage();
+const caseDetailsPage = new CaseDetailsPage();
+
+describe('add case notes', () => {
+	beforeEach(() => {
+		cy.login(users.appeals.caseAdmin);
+	});
+
+	it('add a case note of more than 300 characters', () => {
+		let text = Cypress._.repeat('Here is a case note that is more than 300 characters ', 6);
```

Comment:
Nice use of repeat to keep things consise. I would suggest considering adding note to the end of this line along the lines of `// Text is 53 characters X 6 = 318`

Just to make it very clear what's going on

Reply:
note now added - thank you